### PR TITLE
chore(release): ability to release a beta from develop

### PR DESCRIPTION
**Pull request by bobylito**
This commit adds a new flag to the release command to be able to force a
new beta version from develop.

`npm run release -- --beta`

Remaining tasks:
 - [x] Update contributing guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -321,17 +321,24 @@ npm run release-maintenance
 
 **Be sure to use `$ npm run` instead of `$ yarn run` to avoid issues**
 
-#### Beta version
+#### Beta version β
 
-Beta version release is available on any branch except `master`, `maintainance` or `develop`. The
+Beta version release is available on any branch except `master`, `maintainance`. The
```
Comment (Haroenv) [inline CONTRIBUTING.md]: I don't understand this sentence

Comment (bobylito) [reply CONTRIBUTING.md]: Can you tell what you understand from this sentence?

Comment (Haroenv) [reply CONTRIBUTING.md]: What the difference between the branches is, why develop needs the flag

Comment (bobylito) [reply CONTRIBUTING.md]: The difference is that the default behaviour with develop is to do a complete release.

**Pull request merged**